src/proxy.js

```diff
@@ -75,14 +76,38 @@ function installSensors(router, keycloak) {
                                                                                        proxy(req => mongoProxy.postSensorMeasMongo(       req.params.domain, req.params.sensorID, req.body), true));
   router.get(    '/domains/:domain/sensors/:sensorID/measurements/:measID',            proxy(req => orionProxy.getSensorMeasurement(      req.params.domain, req.params.sensorID, req.params.measID), true));
   router.delete( '/domains/:domain/sensors/:sensorID/measurements/:measID',            proxy(req => orionProxy.deleteSensorMeasurement(   req.params.domain, req.params.sensorID, req.params.measID)),
-                                                                                       proxy(req => mongoProxy.deleteMeasMongo(           req.params.domain, req.params.sensorID, req.params.measID), true));
+                                                                                       proxy(req => mongoProxy.deleteMeasMongo(           req.params.domain, req.params.sensorID, "SensingDevice", req.params.measID), true));

```

Comment:
@cdupont Thats the typo: 'deleteEntityMeasMongo' not 'deleteMeasMongo'. I will fix it soon.